src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -132,115 +230,865 @@ static DamageTable D_8092C140 = {
     /* Powder Keg     */ DMG_ENTRY(1, 0x0),
 };
 
-// sColChkInfoInit
-static CollisionCheckInfoInit D_8092C160 = { 18, 25, 80, MASS_HEAVY };
+static CollisionCheckInfoInit sColChkInfoInit = { 18, 25, 80, MASS_HEAVY };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_8092C168[] = {
+static InitChainEntry sInitChain[] = {
     ICHAIN_F32(targetArrowOffset, 2916, ICHAIN_CONTINUE),
     ICHAIN_VEC3F_DIV1000(scale, 12, ICHAIN_CONTINUE),
     ICHAIN_F32_DIV1000(gravity, -1000, ICHAIN_STOP),
 };
 
-#endif
+static EffectBlureInit2 sBlureInit = {
+    0, 8, 0, { 255, 255, 150, 200 }, { 255, 255, 255, 64 }, { 255, 255, 150, 0 }, { 255, 255, 255, 0 }, 8,
+    0, 2, 0, { 0, 0, 0, 0 },         { 0, 0, 0, 0 },
+};
+
+static s32 texturesDesegmented = 0;
+
+Vec3f sEffectVelAndAccel = { 0.0f, 0.5f, 0.0f };
+
+s8 D_8092C1A8[] = { -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 0,  -1, -1, -1,
+                    -1, -1, -1, -1, -1, -1, 3,  5,  4,  6,  1,  2,  -1, -1, 0,  0 };
+
+Vec3f D_8092C1C8 = { 2000.0f, -200.0f, -5200.0f };
+
+Vec3f D_8092C1D4 = { 300.0f, -200.0f, 0.0f };
+
+s8 D_8092C1E0[] = { -1, -1, -1, 0,  1, 2,  3,  4,  5,  -1, -1, -1, 6,  -1, -1, 7,
+                    -1, -1, 8,  -1, 9, 10, -1, 11, -1, 12, -1, -1, -1, -1, 0,  0 };
+
+Vec3f D_8092C200 = { 0.53f, 0.53f, 0.53f };
```

Comment:
Looks like these 4 can all be in-function static in the functions that use them.

Reply:
In function static works for only one of these, the rest will not match.